doc/modelling/unit_operations/population_balance_model.rst

```diff
@@ -0,0 +1,154 @@
+.. _pbm_model:
+
+Population balance model (PBM)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The population balance equation is a particle-number continuity equation which describes the evolution of the number density :math:`n` of the particles in the time and space domains. 
+The particles of interest have both internal and external coordinates: the internal coordinate can be chosen as any property of the particles such as the particle size or volume 
```

Comment:
This is mathematically true but I would make it more clear. It's not really a spatial coordinate, rather it's the "bin" of the particles. Consider adding a figure to clarify.

Reply:
This is also an implementation-related issue right? Most people don't understand what the bin is, and the bin is also a discretized result, the PBM itself is continuous. The concepts of internal and external coordinate are well-accepted. I think we don't need to mention anything about the discretization details in the document.

Reply:
> This is also an implementation-related issue right? 

Haha, no, it's just a preference because I feel that the notion of "internal coordinate" can be more confusing than helpful but if it's accepted, I will stop complaining.

You're right about the bin/discretization, I was not very precise there. Still, a figure displaying a typical PB might help the reader.

Reply:
In my opinion the PBM module appears somewhat different and inbetween transport and reaction (at least) in two aspects: 1) the internal coordinate is physically different from a spatial location (even though the mathematics of discretization are quite similar) and 2) the current implementation is closer to a reaction module than to a transport module. However, the latter can (and might be) changed in the future.

In conclusion, I would recommed documenting it as a third type of module, different from both transport and reaction type of modules.